api/_globals/createImageBitmap.json

```
@@ -276,7 +276,7 @@
             "opera": "mirror",
             "opera_android": "mirror",
             "safari": {
-              "version_added": false
+              "version_added": "15"
```

I see this committed in August 2023. So version 17 seems more likely to me. https://github.com/WebKit/WebKit/commit/7873a1e800d01e459f642ebba2a8044d587ec867

```suggestion
              "version_added": "17"
```